src/cephadm/tests/test_cephadm.py

```diff
@@ -1981,3 +1975,20 @@ def test_subnet_mask_missing(self):
     def test_subnet_mask_junk(self):
         rc, v, msg = cd.check_subnet('wah')
         assert rc == 1 and msg
+
+    def test_ip_in_subnet(self):
```

Comment:
can we also get some ipv6 test cases here?

Reply:
Added new test cases for IPv6